flink/v1.15/flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkSourceConfig.java

```diff
@@ -31,14 +31,13 @@ public class TestFlinkSourceConfig extends TestFlinkTableSource {
   @Test
   public void testFlinkSessionConfig() {
     getTableEnv().getConfig().set(FlinkReadOptions.STREAMING_OPTION, true);
-    AssertHelpers.assertThrows(
-        "Should throw exception because of cannot set snapshot-id option for streaming reader",
-        IllegalArgumentException.class,
-        "Cannot set as-of-timestamp option for streaming reader",
-        () -> {
-          sql("SELECT * FROM %s /*+ OPTIONS('as-of-timestamp'='1')*/", TABLE);
-          return null;
-        });
+    Assertions.assertThatThrownBy(
+            () -> {
+              sql("SELECT * FROM %s /*+ OPTIONS('as-of-timestamp'='1')*/", TABLE);
+            })
+        .as("Should throw exception because of cannot set snapshot-id option for streaming reader")
```

Comment:
not needed. Please re-visit all `.as()` usage in this PR. I believe most are not needed

Reply:
Addressed for most of the cases